### DISCUSSION TITLE
[has fix] No internet when migrating from _CNI_ to _netavark_?

### DISCUSSION THREAD

**Discussion by tobwen** [containers/podman#17498] [Show and tell]
## issue
I've migrated from _CNI_ to _netavark_ and none of my container had working internet.

## solution
Check if there are old _CNI_ links - _netavark_ isn't compatible with it. After removing it, _netavark_ could/should work without an issue.

```
$ ip link
[...]
4: cni-podman0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN group default qlen 1000
    link/ether 9e:0c:af:02:88:65 brd ff:ff:ff:ff:ff:ff
    inet 10.88.0.1/16 brd 10.88.255.255 scope global cni-podman0
       valid_lft forever preferred_lft forever
    inet6 fe80::9c0c:afff:fe02:8865/64 scope link
       valid_lft forever preferred_lft forever

$ ip link delete cni-podman0
```

**Comment by Luap99** [comment]
I would recommend to reboot, there are also iptables rule from cni that can conflict. A reboot will ensure a clean system.

**Reply by tobwen** [reply]
Indeed. This was just a quick help if anyone is stuck like me.

Rebooting my server is pretty complicated because of all the running VMs on there.
